src/Proto3/Suite/DotProto/Generate/Syntax.hs

```diff
@@ -490,29 +426,27 @@ unbangedTy_ :: HsType -> HsBangType
 unbangedTy_ = noLocA . GHC.HsBangTy synDef (HsSrcBang synDef NoSrcUnpack NoSrcStrict) . parenTy
 
 module_ :: ModuleName -> Maybe [HsExportSpec] -> [HsImportDecl] -> [HsDecl] -> GHC.HsModule
-#if MIN_VERSION_ghc(9,6,0)
+#if MIN_VERSION_ghc_lib_parser(9,6,0)
                                                                                             GhcPs
 #endif
 module_ moduleName maybeExports imports decls = GHC.HsModule
   {
-#if MIN_VERSION_ghc(9,6,0)
+#if MIN_VERSION_ghc_lib_parser(9,6,0)
     hsmodExt = XModulePs
       { hsmodAnn = synDef
       , hsmodLayout = VirtualBraces 2
       , hsmodDeprecMessage = Nothing
       , hsmodHaddockModHeader = Nothing
       }
 #else
-#if MIN_VERSION_ghc(9,2,0)
-    hsmodAnn = synDef,
-#endif
-    hsmodLayout = VirtualBraces 2
+  , hsmodAnn = synDef
+  , hsmodLayout = VirtualBraces 2
```

Comment:
I think the commas are in the wrong place here.

Reply:
I believe the comma issue was due to the confusing CPP branches.
I've refactored the code to simplify the conditionals,
which should resolve this problem.